unit_test/entities/test_production_cluster.py

```diff
@@ -85,11 +84,11 @@ def test_production_cluster_set_setpoints(self) -> None:
         self.assertEqual(self.production_cluster.temperature_return, 333.15)
         self.assertEqual(self.production_cluster.controlled_mass_flow, mass_flow)
         self.assertEqual(
-            self.production_cluster.solver_asset.mass_flow_rate_set_point,
+            self.production_cluster.solver_asset.mass_flow_rate_set_point,  # type: ignore
             self.production_cluster.controlled_mass_flow,
         )
         self.assertNotEquals(
```

Comment:
This should be converted to assertNotEqual

![alt text](image.png)

Reply:
done